src/IO/H5/VolumeData.cpp

```diff
@@ -205,6 +207,98 @@ void VolumeData::write_volume_data(
               append_element_extents_and_connectivity(
                   &total_extents, &total_connectivity, &total_points_so_far,
                   dim, element);
+
+              // If element is 2D and the bases are both SphericalHarmonic,
+              // then add extra connections to close the surface.
```

Comment:
Should this be done here or in GenerateXdmf? It needs only information that's already written out, right?